docker-compose.yml

```diff
@@ -126,13 +126,7 @@ services:
       - share_network
 
   postgres:
-    image: postgres:10
-    command:
-      - /bin/bash
-      - -c
-      - echo "$$POSTGRES_INITDB" > /docker-entrypoint-initdb.d/commands.sh &&
```

Comment:
i'm ok with removing this (i doubt it's needed) but pls also remove the (now unused) `POSTGRES_INITDB` env var